pkgs/development/tools/scenic-view/default.nix

```diff
@@ -81,14 +30,14 @@ in stdenv.mkDerivation rec {
   inherit pname version src;
   nativeBuildInputs = [ jdk gradle makeWrapper ];
```

Comment:
How does this work exactly; doesn't gradle bring its own JDK?

Reply:
I simply ported the previous derivation, I'll check if this works without jdk later

Reply:
Yes, of course. I was merely curious how that works or what you think is how it's supposed to work.

Reply:
the simple answer is it isn't, if the intent was to make Gradle use a specific Java, it should be passed as `java.home`.